### DISCUSSION TITLE
Include a prior FIM

### DISCUSSION THREAD

**Discussion by erikmhahn** [andrewhooker/PopED#62] [Q&A] [answered]
There is an example for including a prior FIM using popED (https://cran.r-project.org/web/packages/PopED/vignettes/examples.html#include-a-prior-fim-compute-power-to-identify-a-parameter). 
In this example, it is noted that "one could also use the observed FIM when using estimation software to fit one realization of a design (from the $COVARIANCE step in NONMEM for example)."
I was wondering how you would implement this in popED?

**Comment by andrewhooker** [comment, marked as answer]
Hi Erik,

The process would look like this:

1. read in the covariance matrix of NONMEM into R (from the *.cov file is easiest)
2. rearrange the that matrix to match the order of parameters seen in the FIM calculated in PopED (potentially remove rows and columns from the matrix if extra parameters were estimated in NONMEM, or if there are rows of zeros, which sometimes occur in NONMEM *.cov files)
3. Use this matrix as the prior instead of one computed from PopED

This method uses, as a prior, the uncertainty of the realized design (from NONMEM) instead of the expected uncertainty of a future experiment as a prior (as you would get from PopED evaluating a design).  So there will be differences depending on the observed data.

Best regards,
Andy
